playbooks/cluster/openshift/config.yml

```diff
@@ -1,13 +1,15 @@
 ---
 - hosts: all
   remote_user: root
+  vars:
+    epel_release_rpm_url: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
@rthallisey I'm not an Ansible expert, and I don't follow why it's here if there is the same var in playbooks/cluster/openshift/defaults/main.yaml ?

Reply:
Something I noticed in CI was that the variables in defaults are not being used.  But when I run the playbook locally they do.  So to answer your first question, I tried changing the name from yml to yaml, but it didn't work.  Finally I added the variable here for now to see if CI will go back to green and it did.

I'll investigate this some more tomorrow.